src/elements/content-sidebar/activity-feed/comment/Comment.js

```diff
@@ -166,7 +166,11 @@ class Comment extends React.Component<Props, State> {
                                 }
                             >
                                 <small className="bcs-comment-created-at">
-                                    <ReadableTime alwaysShowTime timestamp={createdAtTimestamp} />
+                                    <ReadableTime
+                                        relativeThreshold={-1}
```

Comment:
What does this do and why is it needed? I'm not seeing any similar usages in the codebase.

Reply:
Instead of "5s ago" it shows "Today at 5:28pm", ie a relative day but not time.

Reply:
Should it say `relativeThreshhold={0}`, in that case?

Reply:
That should work as well